Kitodo/src/test/java/org/kitodo/services/data/BatchServiceIT.java

```diff
@@ -36,8 +37,9 @@ public static void prepareDatabase() throws DAOException, IOException, ResponseE
     }
 
     @AfterClass
-    public static void cleanDatabase() {
+    public static void cleanDatabaseAndIndex() throws IOException, ResponseException{
```

Comment:
Fixed, now method is only used by classes which currently make operations on ElasticSearch index.